common/lib/user.js

```diff
@@ -40,7 +40,9 @@ User.prototype = {
       permissions.push(
         'moves:view:by_location',
         'moves:download:by_location',
-        'move:view'
+        'move:view',
```

Comment:
Prison users can now move people. Figured if you can create you should also be able to delete?